SECURITY.md

```diff
@@ -22,9 +22,11 @@ Alternately, send an email to `security@tc39.es`
 > [!NOTE]
 > This list is not exhaustive.
 
-| Engine/Platform | Used In                | Link to Report                                  |
-| --------------- | ---------------------- | ----------------------------------------------- |
-| JavaScriptCore  | Safari                 | [Report](https://webkit.org/security-policy/)   |
-| Node            |                        | [Report](https://nodejs.dev/en/about/security/) |
-| SpiderMonkey    | Firefox                | [Report](https://www.mozilla.org/security/)     |
-| V8              | Chrome, Chromium, Edge | [Report](https://v8.dev/docs/security-bugs)     |
+| Engine/Runtime | Used In                            | Link to Report                                             |
+| -------------- | ---------------------------------- | ---------------------------------------------------------- |
+| JavaScriptCore | Safari, Bun                        | [Report](https://webkit.org/security-policy/)              |
+| SpiderMonkey   | Firefox                            | [Report](https://www.mozilla.org/security/)                |
+| V8             | Chrome, Chromium, Edge, Node, Deno | [Report](https://v8.dev/docs/security-bugs)                |
+| Node           |                                    | [Report](https://nodejs.dev/en/about/security/)            |
```

Comment:
```suggestion
| Chakra         | Edge < 80                       | [Report](https://github.com/chakra-core/ChakraCore?tab=readme-ov-file#security) |
| Node           |                                    | [Report](https://nodejs.dev/en/about/security/)            |
```

Reply:
security fixes only till 2021 though

Reply:
last release was in 2020

Reply:
that's just from microsoft; the community continues to maintain and support it.